core/trino-main/src/main/java/io/trino/sql/planner/LogicalPlanner.java

```diff
@@ -591,11 +598,16 @@ private RelationPlan getInsertPlan(
         TableStatisticsMetadata statisticsMetadata = metadata.getStatisticsCollectionMetadataForWrite(session, tableHandle.catalogHandle(), tableMetadata.metadata());
 
         if (materializedViewRefreshWriterTarget.isPresent()) {
+            WriterTarget writerTarget = materializedViewRefreshWriterTarget.get();
```

Comment:
You can make `materializedViewRefreshWriterTarget` to have `RefreshMaterializedViewReference` type, so if is not needed